php/plan.sh

```diff
@@ -33,6 +39,9 @@ pkg_include_dirs=(include)
 pkg_interpreters=(bin/php)
 
 do_build() {
+
+  set_ld_library_path
+
   ./configure --prefix="${pkg_prefix}" \
```

Comment:
Seeing LD_LIBRARY_PATH always makes me nervous.  Looking into why it's necessary, the configuration testing doesn't use the LD_RUN_PATH env var, so the resulting binary produced isn't runnable.   Short of patching the build process, which would be a lot more work, I agree with the need for LD_LIBRARY_PATH here, but I'd like to limit the scope of its use. 

The logs indicate that the missing library is libstdc++.so, which is provided by core/gcc-libs, and should only be needed during the `./configure`.  Rather than set LD_LIBRAY_PATH globally, with all dependencies,  how about setting it explicitly to the needed library, and only during `./configure`

```suggestion
  # The configuration scripts unset LD_RUN_PATH when testing linking for configured options,
  # so the resulting 'conftest' binaries cannot run due to being unable to find libstdc++.so 
  # This allows those binaries to execute while limiting the scope of LD_LIBRARY_PATH 
  # to the execution of `./configure`.
  LD_LIBRARY_PATH="$(pkg_path_for core/gcc-libs)" ./configure --prefix="${pkg_prefix}" \
```

Reply:
@smacfarlane thanks good idea - had to adapt it a little bit (we needed gcc)

also nice explanation...